langchain/src/vectorstores/supabase.ts

```diff
@@ -36,16 +45,20 @@ export interface SupabaseLibArgs {
   client: SupabaseClient;
   tableName?: string;
   queryName?: string;
+  queryWithEmbeddingsName?: string;
```

Comment:
What's this for?

Reply:
Oops, that was left over from a previous iteration I did, where there would be two separate Postgres functions, one with and one without the embeddings in the return values. Removed in [1507a0d](https://github.com/langchain-ai/langchainjs/pull/2800/commits/1507a0dcbb5713e9ef159ecfffb31ad325947dd8)